### PR TITLE
rename colorscheme to set-color-scheme and use color-schemes dir

### PR DESCRIPTION
**Pull request by TeddyDD**
`:set-color-scheme` instead `:colorscheme` (alias `color-scheme`)
`color-schemes` directory instead `colors`

Closes: #1733

### CODE REVIEW

File: README.asciidoc [hunk 1]
```diff
@@ -679,7 +679,7 @@ Color Schemes
 
 Kakoune ships with some color schemes that are installed to
 `../share/kak/colors/`. If `$XDG_CONFIG_HOME/kak/colors/` is present
```
Comment (Screwtapello) [inline README.asciidoc]: You need to change these paths too!

Comment (TeddyDD) [reply README.asciidoc]: Good catch

**Comment by TeddyDD** [comment]
umm, failing tests are not good sign :thinking:

**Comment by TeddyDD** [comment]
Fixed

**Comment by mawww** [comment]
I am wondering if `set` is the correct verb here, I guess it makes sense as color schemes are mostly going to run `set-face` commands. Any feedback from the community on this change ?

**Comment by alexherbo2** [comment]
`apply`?

**Comment by lenormf** [comment]
I've commented otherwise in the related issue, but now I'm thinking there's no real benefit to renaming that command.

**Pull request closed without merging**